config.json

```diff
@@ -52,14 +52,17 @@
                 {
                     "name":"Proguard Guidelines",
                     "platforms": ["Android"]
-
                 },
                 {
                     "name": "Deploying to AWS",
                     "platforms": ["JavaEE"]
                 },
                 {
                     "name": "Example Apps"
+                },
+                {
+                    "name": "React Native",
```

Comment:
Change to "Can I Integrate SDL into a React Native App?" and move to the FAQ section.